specification/cluster/_types/ClusterStateBlocksIndex.ts

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-import { IndicesRolloverConditions } from '@indices/rollover/IndicesRolloverConditions'
+import { RolloverConditions } from '@indices/rollover/types'
```

Comment:
shouldn't `RolloverConditions` being in `indices/_types` if it's used in more than one endpoint?

Reply:
It's used only there (and cluster) :)